File: java/Clothing.java
package LP2.java;

public class Clothing extends Product {
    // varible private
    private String size;
    private String material;
    private char gender;

    // constructor
    public Clothing(){
        this.size = "";
        this.material = "";
        this.gender = '-';
    }

    public Clothing(String size, String material, char gender) {
        this.size = size;
        this.material = material;
        this.gender = gender;
    }

    // setter dan getter
    public String getSize(){
        return this.size;
    }

    public void setSize(String size){
        this.size = size;
    }

    public String getMaterial(){
        return this.material;
    }

    public void setMaterial(String material){
        this.material = material;
    }

    public char getGender(){
        return this.gender;
    }

    public void setGender(char gender){
        this.gender = gender;
    }
}

File: python/Program/Clothing.py
from Product import Product  # Assuming you have a product.py file with the Product class

class Clothing(Product):
    def __init__(self):
        super().__init__()
        self.__size = ""
        self.__material = ""
        self.__gender = ""

    def set_size(self, size):
        self.__size = size

    def get_size(self):
        return self.__size

    def set_material(self, material):
        self.__material = material

    def get_material(self):
        return self.__material

    def set_gender(self, gender):
        self.__gender = gender

    def get_gender(self):
        return self.__gender